Composer/packages/client/src/pages/design/creationModal.tsx

```diff
@@ -66,7 +65,7 @@ export const CreationModal: React.FC<CreationModalProps> = (props) => {
 
   const fetchResources = async () => {
     await fetchStorages();
-    fetchTemplates();
+    fetchTemplatesV2();
```

Comment:
the v2 is because we haven't removed the legacy code yet correct?

Reply:
yes correct. Right now we are prioritizing clean up the scenarios that will be tested by the larger audience during next weeks bug bash. Its on our backlog to clean up and remove loads of legacy code prior to release.